server/src/internalClusterTest/java/org/elasticsearch/repositories/blobstore/BlobStoreRepositoryCleanupIT.java

```diff
@@ -116,11 +112,8 @@ public void testCleanupOldIndexN() throws ExecutionException, InterruptedExcepti
         internalCluster().startNodes(Settings.EMPTY);
 
         final String repoName = "test-repo";
-        logger.info("-->  creating repository");
-        assertAcked(client().admin().cluster().preparePutRepository(repoName).setType("fs").setSettings(Settings.builder()
-            .put("location", randomRepoPath())
-            .put("compress", randomBoolean())
-            .put("chunk_size", randomIntBetween(100, 1000), ByteSizeUnit.BYTES)));
+        createRepository(repoName, "fs", Settings.builder().put("location", randomRepoPath()).put("compress", randomBoolean())
```

Comment:
I think it deserves a `createFsRepository()`; I'd also rarely set a random chunk size

Reply:
Let's do this in a follow up since it's so many spots to change? I was thinking about unifying the various random settings we use as much as possible anyway. Currently there's so many different kinds of random which aren't really needed in most cases. There's just a handful of tests where things like `compression` -> `false` are actually necessary and the rest could all just use the same randomness. Just figured that's probably best done separately since it's a logical change to the tests?

Reply:
Sure